qube/create/templates/common_files/{{ cookiecutter.commonName }}/.github/workflows/pr_to_master_from_patch_release_only.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       # PRs to the repository master branch are only ok if coming from any patch or release branch
       - name: Check PRs
         run: |
-          { [[ $GITHUB_HEAD_REF = *"release"* ]]; } || [[ $GITHUB_HEAD_REF == *"patch"* ]]
+          { [[ $GITHUB_HEAD_REF = *"release"* ]]; } || [[ $GITHUB_HEAD_REF == *"hotfix"* ]]
```

Comment:
Then the posted message has to be updated as well:

> Because of this, PRs to `master` are only allowed if they come from any ${{github.event.pull_request.head.repo.full_name}} `release` or `patch` branch.

Reply:
-> 
`Because of this, PRs to `master` are only allowed if they come from any ${{github.event.pull_request.head.repo.full_name}} `release` or `hotfix` branch.`

Reply:
Yes!